tests/backend/user/test_apiv1_human_find.py

```diff
@@ -62,7 +62,7 @@ async def test_isolation_from_other_organization(
     backend, alice, sock_from_other_organization_factory
 ):
     async with sock_from_other_organization_factory(backend) as sock:
-        rep = await human_find(sock, query=alice.user_id)
+        rep = await human_find(sock, query=alice.human_handle.label)
```

Comment:
You should also add a call to human_find without `query` param to make sure we only have users from our ow organization this way

Reply:
And this test also exists in `test_human_find.py` (i.e. the apiv2 version), it should be corrected as well

Reply:
Not sure but it should be done with lines below, I assert that users from alice_backend_sock are different from the other organization to test the isolation.